src/Core/Bootstrap/ConfigurationLoader.php

```diff
@@ -79,6 +74,21 @@ protected function loadConfigurationFiles(Application $app, RepositoryContract $
         }
     }
 
+    /**
+     * Get the path to the routes cache file.
+     *
+     * @return string
+     */
+    protected function maybeForceWpConfigInclude()
+    {
+        // Avoid duplicate constants definitions.
+        if (defined('AUTH_KEY')) {
```

Comment:
@ogorzalka Could you move this condition into its own method and replace it here as well as inside the `loadConfigurationFiles` method please ?